packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -361,7 +361,7 @@ export default function CollabSidebar() {
 		? resultComments.find( ( thread ) => thread.id === blockCommentId )
 		: null;
 
-	// If postId is not a valid number, do not render the comment sidebar.
+	// If postId is not a valid number, do not render the comment sidebar. 1Code has comments. Press enter to view.
```

Comment:
```suggestion
	// If postId is not a valid number, do not render the comment sidebar.
```

This seems to be leftover.